client/blocks/site/index.jsx

```diff
@@ -165,6 +165,11 @@ class Site extends Component {
 									: translate( 'Private' ) }
 							</span>
 						) }
+						{ site.options && site.options.is_difm_lite_in_progress && (
+							<span className="site__badge site__badge-domain-only">
+								{ translate( 'Do It For Me' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'Do It For Me (Lite)' )` *ES Score: 7*